ReactiveUI/PropertyBinding.cs

```diff
@@ -453,17 +598,112 @@ public class PropertyBinderImplementation : IPropertyBinderImplementation
+        }
+
+        /// <summary>
+        /// Binds the specified view model property to the given view property.
+        /// </summary>
+        /// <typeparam name="TViewModel">The type of the view model being bound.</typeparam>
+        /// <typeparam name="TView">The type of the view being bound.</typeparam>
+        /// <typeparam name="TVMProp">The type of the property bound on the view model.</typeparam>
+        /// <typeparam name="TVProp">The type of the property bound on the view.</typeparam>
+        /// <typeparam name="TDontCare">
+        /// A dummy type, only the fact that <paramref name="signalViewUpdate"/> 
+        /// emits values is considered, not the actual values emitted.
+        /// </typeparam>
+        /// <param name="view">The instance of the view to bind.</param>
+        /// <param name="viewModel">The instance of the view model to bind.</param>
+        /// <param name="vmProperty">
+        /// An expression indicating the property that is bound on the view model.
+        /// This can be a chain of properties of the form <code>vm =&gt; vm.Foo.Bar.Baz</code>
+        /// and the binder will attempt to subscribe to changes on each recursively.
+        /// </param>
+        /// <param name="viewProperty">
+        /// The property on the view that is to be bound.
+        /// This can be a chain of properties of the form <code>view => view.Foo.Bar.Baz</code>
+        /// and the binder will attempt to set the last one each time the view model property is updated.
+        /// </param>
+        /// <param name="signalViewUpdate">
+        /// An observable, that when signaled, indicates that the view property 
+        /// has been changed, and that the binding should update the view model
+        /// property accordingly.
+        /// </param>
+        /// <param name="vmToViewConverter">
+        /// Delegate to convert the value of the view model's property's type to a value of the
+        /// view's property's type.
+        /// </param>
+        /// <param name="viewToVmConverter">
+        /// Delegate to convert the value of the view's property's type to a value of the
+        /// view model's property's type.
+        /// </param>
+        /// <returns>
+        /// An instance of <see cref="IDisposable"/> that, when disposed,
+        /// disconnects the binding.
+        /// </returns>
+        public IReactiveBinding<TView, TViewModel, Tuple<object, bool>> Bind<TViewModel, TView, TVMProp, TVProp, TDontCare>(
+                TViewModel viewModel,
+                TView view,
+                Expression<Func<TViewModel, TVMProp>> vmProperty,
+                Expression<Func<TView, TVProp>> viewProperty,
+                IObservable<TDontCare> signalViewUpdate,
+                Func<TVMProp, TVProp> vmToViewConverter,
+                Func<TVProp, TVMProp> viewToVmConverter)
+            where TViewModel : class
+            where TView : IViewFor
+        {
```

Comment:
For the avoidance of confusion, should we throw `ArgumentNullException` here if either `vmToViewConverter` or `viewToVmConverter` is `null`?